src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -34,4 +45,15 @@ public Command parseCommand(String userInput) {
             return new HelpCommand();
         }
     }
+
+    private Command prepareAdd(String arguments) {
+
+        final int indexOfAtPrefix = arguments.indexOf("i/");
+        final int indexOfAtPrefix_2 = arguments.indexOf("p/");
+        String description = arguments.substring(indexOfAtPrefix, indexOfAtPrefix_2);
```

Comment:
I think this description includes the "i/" inside, just a slight change to the index will do!

Reply:
got it! Thank you ^^